src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -102,12 +113,26 @@ const ViewFieldIngelogdExtraInfo = ({ crudObject, hideEdit, className }) => {
     )
 }
 
+/**
+ * Component that renders the EigenarenList component, which displays the ownerlist.
+ *
+ * @component
+ *
+ * @param {Array} eigenaren - Parameter containing a collection of owners.
```

Comment:
type Object

Reply:
I've changed the parameter description and type to object.